meson.build

```diff
@@ -99,18 +106,31 @@ subdir('plugins')
 
 install_subdir('shaders', install_dir: 'share/wayfire')
 
-summary = [
+wlroots_xwayland_info = []
+
+if use_system_wlroots
+  ccp = meson.get_compiler('cpp')
```

Comment:
Maybe the variable should be named cpp instead of ccp? I think this was my mistake but probably should fix it while it's on the table.

Reply:
done